src/main/java/com/rpg2014/ServerController.java

```diff
@@ -27,7 +28,8 @@
     @Produces(MediaType.APPLICATION_JSON)
     public StatusResponse serverStatus() {
 //        Status status = EC2Wrapper.getServerStatus();
-        return StatusResponse.builder().status(Status.Terminated).build();
+
+        return StatusResponse.builder().status(MinecraftDynamoWrapper.getInstance().isServerRunning()).build();
```

Comment:
Don't commit this change either.  This method will call the isServerRunning method in the ec2 wrapper.